src/__tests__/unusedTranslations.test.ts

```diff
@@ -11,7 +11,12 @@ import en from './../messages/ui.en.json';
 import * as fs from 'fs';
```

Comment:
This change is needed unstream too as this test no longer fails as translations.ts isn't the relevant file to ignore anymore.